src/Chapter1.hs

```diff
@@ -632,9 +646,12 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
-
+sumLast2 :: Int -> Int
+sumLast2 n = last + penultimate  
+  where
+    getLast     = mod n 100
+    last        = mod getLast 10  
```

Comment:
`last` is the name of a standard function. Haskell allows name shadowing, that's why it works. Though, in practice, it's usually better to avoid name clashes with other functions cause it could be confusing 🙂